---
title: "Apps"
---

This page lists known public deployments of [iSEE](https://bioconductor.org/packages/iSEE/) apps.

Links are ordered in reverse order of [iSEE](https://bioconductor.org/packages/iSEE/) package version (i.e., those using the most recent version appear first).

## John et al, bioRxiv, 2025

![John et al, bioRxiv, 2025.](images/apps/john_et_al.png)

{{< fa arrow-pointer >}} Shiny app <https://shiny.mpl.mpg.de/wehner_lab/spinal_cord_regeneration_atlas/>

{{< fa user >}} Authored by [Wehner group](https://mpl.mpg.de/research-at-mpl/independent-research-groups/wehner-research-group)

{{< fa cog >}} Using **iSEE** {2.18.0}, **iSEEhex** {1.8.0}, **iSEEu** {1.18.0}

**More:** This app accompanies the preprint "Biphasic inflammation control by dedifferentiated fibroblasts enables axon regeneration after spinal cord injury in zebrafish" <https://www.biorxiv.org/content/10.1101/2025.01.27.635043v1>.

## Yalcinbas et al., American Journal of Psychiatry, 2025

![Yalcinbas et al., American Journal of Psychiatry, 2025.](images/apps/yalcinbas_et_al.png)

{{< fa arrow-pointer >}} Shiny apps: <https://libd.shinyapps.io/habenulaPilot_bulk/> (bulk RNA-seq), <https://libd.shinyapps.io/habenulaPilot_snRNAseq/> (snRNA-seq)

{{< fa user >}} Authored by [The Lieber Institute for Brain Development](https://www.libd.org/)

{{< fa cog >}} Using **iSEE** {2.12.0}

**More:** This app accompanies the article "Transcriptomic Analysis of the Human Habenula in Schizophrenia" <https://psychiatryonline.org/doi/10.1176/appi.ajp.20240776>.

## Rigby et al., bioRxiv, 2023

![Rigby et al., bioRxiv, 2023.](images/apps/rigby_et_al.png)

{{< fa arrow-pointer >}} Shiny app <https://rehwinkellab.shinyapps.io/ifnresource/>

{{< fa user >}} Authored by [Rehwinkel group](https://www.imm.ox.ac.uk/research/research-groups/rehwinkel-group-nucleic-acid-sensing)

{{< fa cog >}} Using **iSEE** {2.12.0}, **iSEEindex** {0.99.13}, **iSEEde** {0.99.0}

**More:** This app accompanies the preprint "Single-cell analysis of signalling and transcriptional responses to type I interferons" <https://www.biorxiv.org/content/10.1101/2023.07.03.547491v1>.

## FGCZ Fancy Single Cell Explorer

![FGCZ Fancy Single Cell Explorer (Index page).](images/apps/fgcz.png)

{{< fa arrow-pointer >}} Shiny app <https://fgcz-shiny.uzh.ch/fgcz_iSEE/>

{{< fa user >}} Authored by [Functional Genomics Center Zurich (FGCZ)](https://fgcz.ch/)

{{< fa cog >}} Using **iSEE** {2.12.0}

**More:** This app is listed at <https://fgcz-shiny.uzh.ch/connect/>, among other interactive apps from the same author.

## Moffitt et al., Science, 2018

![Moffitt et al., Science, 2018.](images/apps/moffitt_hypothalamus.png)

{{< fa arrow-pointer >}} Shiny app <https://moffittlab.connect.hms.harvard.edu/iSEE_hypothalamus/>

{{< fa user >}} Authored by [Center for Computational Biomedicine (CCB) Harvard Medical School](https://computationalbiomed.hms.harvard.edu/) and [Moffitt Lab](https://moffittlab.github.io/)

{{< fa cog >}} Using **iSEE** {2.10.0}

**More:** This app showcases data from the article "Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region" <https://www.science.org/doi/10.1126/science.aau5324> and is listed at <https://moffittlab.connect.hms.harvard.edu/merfish/merfish_homepage.html>, among other interactive apps from the same author.

## Petukhov et al., Nat Biotech, 2021

![Petukhov et al., Nat Biotech, 2021.](images/apps/moffitt_ileum.png)

{{< fa arrow-pointer >}} Shiny app <https://moffittlab.connect.hms.harvard.edu/iSEE_ileum/>

{{< fa user >}} Authored by [Center for Computational Biomedicine (CCB) Harvard Medical School](https://computationalbiomed.hms.harvard.edu/) and [Moffitt Lab](https://moffittlab.github.io/)

{{< fa cog >}} Using **iSEE** {2.10.0}

**More:** This app showcases data from the article "Cell segmentation in imaging-based spatial transcriptomics" <https://www.nature.com/articles/s41587-021-01044-w> and is listed at <https://moffittlab.connect.hms.harvard.edu/merfish/merfish_homepage.html>, among other interactive apps from the same author.

## Kim et al, JNeurosci, 2023

![Kim et al, JNeurosci, 2023.](images/apps/jns2023.png)

{{< fa arrow-pointer >}} Shiny app <https://hbcc-nimh.shinyapps.io/shinyApp_JNS2023/>

{{< fa user >}} Authored by [Human Brain Collection Core (HBCC) - NIMH](https://www.nimh.nih.gov/research/research-conducted-at-nimh/research-areas/research-support-services/hbcc)

{{< fa cog >}} Using **iSEE** {2.10.0}

**More:** This app accompanies the  "Cellular Diversity in Human Subgenual Anterior Cingulate
and Dorsolateral Prefrontal Cortex by Single-Nucleus RNASequencing" <https://doi.org/10.1523/jneurosci.0830-22.2023>.

## Kwon et al., bioRxiv, 2023

![Kwon et al., bioRxiv, 2023.](images/apps/kwon_et_al_biorxiv_2023.png)

This app showcases spatially resolved transcriptomics of the inferior temporal cortex, as pseudobulks.

{{< fa arrow-pointer >}} Shiny app <https://libd.shinyapps.io/Visium_SPG_AD_pseudobulk_AD_pathology_wholegenome/>

{{< fa user >}} Authored by [The Lieber Institute for Brain Development](https://www.libd.org/)

{{< fa cog >}} Using **iSEE** {2.10.0}

**More:** This app accompanies the preprint "Influence of Alzheimer’s disease related neuropathology on local microenvironment gene expression in the human inferior temporal cortex" <https://doi.org/10.1101/2023.04.20.537710>.

## iSEE COVID Hub

![iSEE COVID Hub (Monocytes).](images/apps/IMBEI_monocytes.png)

Shiny apps:

{{< fa arrow-pointer >}} <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Bcells/> (B cells)

{{< fa arrow-pointer >}} <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Monocytes> (Monocytes)

{{< fa arrow-pointer >}} <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_NKcells/> (NK cells)

{{< fa arrow-pointer >}} <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Tcells> (T cells)

{{< fa user >}} Authored by [Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI)](https://www.unimedizin-mainz.de/imbei/imbei/welcome-page.html?L=1)

{{< fa cog >}} Using **iSEE** {2.9.0}

**More:** Those apps are listed at <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT/>.

## Billing et al., Circulation, 2023

![Billing et al., Circulation, 2023](images/apps/sglt2i_proteome.png)

Shiny apps:

{{< fa arrow-pointer >}} <https://kidneyapp.shinyapps.io/SGLT2i_proteome/> (proteome)

{{< fa arrow-pointer >}} <https://kidneyapp.shinyapps.io/SGLT2i_phosphoproteome/> (phosphoproteome)

{{< fa user >}} Authored by Vallon and Rinschen labs (La Jolla/Aarhus)

{{< fa cog >}} Using **iSEE** {2.8.0}, **iSEEu** {1.8.0}

**More:** A set of proteomic and phosphoproteomic datasets containing measurements for 5 organs (kidney, liver, heart, skeletal muscle, white adipose tissue (WAT)) of 2 mouse strains. Accompanies the publication [Metabolic Communication by SGLT2 Inhibition](https://www.ahajournals.org/doi/full/10.1161/CIRCULATIONAHA.123.065517).

## Kalucka et al., Cell, 2020

![Kalucka et al., Cell, 2020.](images/apps/vonandrian_Kalucka20.png)

{{< fa arrow-pointer >}} Shiny app <https://vonandrian.connect.hms.harvard.edu/iSEE_Kalucka20_scECAtlas/>

{{< fa user >}} Authored by [von Andrian Laboratory](https://vonandrian.hms.harvard.edu/)

{{< fa cog >}} Using **iSEE** {2.8.0}, **iSEEu** {1.5.2}

**More:** This app showcases data from the article "Single-cell transcriptome atlas of murine endothelial cells" <https://doi.org/10.1016/j.cell.2020.01.015>.

## Brulois et al., Nat Commun, 2020

![Brulois et al., Nat Commun, 2020.](images/apps/vonandrian_Brulois20.png)

{{< fa arrow-pointer >}} Shiny app <https://vonandrian.connect.hms.harvard.edu/iSEE_Brulois20_PLN/>

{{< fa user >}} Authored by [von Andrian Laboratory](https://vonandrian.hms.harvard.edu/)

{{< fa cog >}} Using **iSEE** {2.8.0}, **iSEEu** {1.5.2}

**More:** This app showcases data from the article "A molecular map of murine lymph node blood vascular endothelium at single cell resolution" <https://doi.org/10.1038/s41467-020-17291-5>.

## Myocyte data sets

![Myocyte data sets (Index page).](images/apps/MaxDelbruck_MyoExplorer.png)

{{< fa arrow-pointer >}} Shiny app <https://shiny.mdc-berlin.de/MyoExplorer/>

{{< fa user >}} Authored by [Max Delbrück Center](https://www.mdc-berlin.de/)

{{< fa cog >}} Using **iSEE** {2.6.0}

## Upload your own SummarizedExperiment

![Upload your own SummarizedExperiment (Index page).](images/apps/ccb_enteprise.png)

This app allows visitors to upload an RDS file that contains a [SummarizedExperiment][summarizedexperiment-bioc] object and another optional RDS file that contains an initial configuration object.

{{< fa arrow-pointer >}} Shiny app <https://ccb-rstudio-connect.hms.harvard.edu/iSEE_enterprise/>

{{< fa user >}} Authored by [Center for Computational Biomedicine (CCB) Harvard Medical School](https://computationalbiomed.hms.harvard.edu/)

{{< fa cog >}} Using **iSEE** {2.4.0}

## Tran et al., Neuron, 2021

![Tran et al., Neuron, 2021 (AMY region).](images/apps/LIBD_tran2021_AMY.png)

Shiny apps:

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2020_Amyg/>

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2021_NAc/>

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2021_sACC/>

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2021_DLPFC/>

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2021_AMY/>

{{< fa arrow-pointer >}} <https://libd.shinyapps.io/tran2021_HPC/>

{{< fa user >}} Authored by [The Lieber Institute for Brain Development](https://www.libd.org/)

{{< fa cog >}} Using **iSEE** {2.4.0}

**More:** Those apps accompany the article "Single-nucleus transcriptome analysis reveals cell-type-specific molecular signatures across reward circuitry in the human brain" <https://www.cell.com/neuron/fulltext/S0896-6273(21)00655-3>.

## Cristinelli et al., Front. Virol., 2021

![Cristinelli et al., Front. Virol., 2021](images/apps/HITEAM.png)

The authors developed HI-TEAM (HIV-Infected cell Transcriptome and EpitrAnscriptoMe), a user-friendly querying platform based on the iSEE interactive interface [iSEE] to explore all data or any gene of interest.
More information is Shiny app <http://sib-pc17.unil.ch/HIVmain.html>.

{{< fa arrow-pointer >}} Shiny app <https://hi-team.unil.ch/#>

{{< fa user >}} Authored by [Ciuffi Lab](https://www.chuv.ch/en/microbiologie/imu-home/research/research-groups/angela-ciuffi)

{{< fa cog >}} Using **iSEE** {2.1.10}, **iSEEu** {1.1.4}

**More:** This app accompanies the article "Single-nucleus transcriptome analysis reveals cell-type-specific molecular signatures across reward circuitry in the human brain" <https://www.frontiersin.org/articles/10.3389/fviro.2021.714475/full>.

## Comparison of SARS-CoV-1 and SARS-CoV-2

![Comparison of SARS-CoV-1 and SARS-CoV-2.](images/apps/MDC_COV.png)

{{< fa arrow-pointer >}} Shiny app <https://shiny.mdc-berlin.de/COV/>

{{< fa user >}} Authored by [Max Delbrück Center](https://www.mdc-berlin.de/)

{{< fa cog >}} Using **iSEE** {1.6.1}

## CD8 TILs from B16 melanoma tumors

![CD8 TILs from B16 melanoma tumors.](images/apps/tilatlas_B16_CD8TIL_10X.png)

{{< fa arrow-pointer >}} Shiny app <https://tilatlas.shinyapps.io/B16_CD8TIL_10X/>

{{< fa user >}} Authored by [Swiss Portal for Immune Cell Analysis](https://spica.unil.ch/)

{{< fa cog >}} Using **iSEE** {1.2.4}

## Demonstration iSEE apps (2018)

![Demonstration iSEE apps (CyTOF data set).](images/apps/iSEE_cytof_2018.png)

Shiny apps:

{{< fa arrow-pointer >}} <https://marionilab.cruk.cam.ac.uk/iSEE_allen> (single-cell RNA-seq, mouse visual cortex)

{{< fa arrow-pointer >}} <https://marionilab.cruk.cam.ac.uk/iSEE_tcga> (RNA-seq, The Cancer Genome Atlas)

{{< fa arrow-pointer >}} <https://marionilab.cruk.cam.ac.uk/iSEE_pbmc4k> (single-cell RNA-seq, peripheral blood mononuclear cells)

{{< fa arrow-pointer >}} <https://marionilab.cruk.cam.ac.uk/iSEE_cytof> (mass cytometry, healthy and diseased human donors)

{{< fa user >}} Authored by [Marioni group](https://www.cruk.cam.ac.uk/research-groups/marioni-group)

{{< fa cog >}} Using **iSEE** {1.2.4}

**More:** Those apps are listed at <https://marionilab.cruk.cam.ac.uk/>, among other interactive apps from the same author.

[iseeindex-bioc]: https://bioconductor.org/packages/devel/bioc/html/iSEEindex.html
[summarizedexperiment-bioc]: https://bioconductor.org/packages/SummarizedExperiment/
