---
title: "Gallery"
---

This page lists known public deployments of [iSEE](https://bioconductor.org/packages/iSEE/) apps.

Links are ordered in reverse order of [iSEE](https://bioconductor.org/packages/iSEE/) package version (i.e., those using the most recent version appear first).

## Collection of single-cell data sets using iSEEindex

- **URL:** <https://rehwinkellab.shinyapps.io/ifnresource/>
- **Maintainer:** [Rehwinkel group](https://www.imm.ox.ac.uk/research/research-groups/rehwinkel-group-nucleic-acid-sensing)
- **Version:** iSEE {2.12.0}, iSEEindex {0.99.13}, iSEEde {0.99.0}

**More:** This app accompanies the preprint [Single-cell analysis of signalling and transcriptional responses to type I interferons](https://www.biorxiv.org/content/10.1101/2023.07.03.547491v1).

## iSEEindex demo

- **URL:** <https://isee.shinyapps.io/iSEEindex-demo/>
- **Maintainer:** [iSEE developers](https://github.com/iSEE)
- **Version:** iSEE {2.12.0}, iSEEindex {0.99.8}

## FGCZ Fancy Single Cell Explorer

- **URL:** <https://fgcz-shiny.uzh.ch/fgcz_iSEE/>
- **Maintainer:** [Functional Genomics Center Zurich (FGCZ)](https://fgcz.ch/)
- **Version:** iSEE {2.12.0}

**More:** This app is listed at <https://fgcz-shiny.uzh.ch/connect/>, among other interactive apps from the same developer.

## MERFISH mouse data sets

- **URL (hypothalamus):** <https://moffittlab.connect.hms.harvard.edu/iSEE_hypothalamus/>
- **URL (ileum):** <https://moffittlab.connect.hms.harvard.edu/iSEE_ileum/>
- **Maintainer:** [Center for Computational Biomedicine (CCB) Harvard Medical School](https://computationalbiomed.hms.harvard.edu/) and [Moffitt Lab](https://moffittlab.github.io/)
- **Version:** iSEE {2.10.0}

**More:** Those apps are listed at <https://moffittlab.connect.hms.harvard.edu/merfish/merfish_homepage.html>.

## iSEE COVID Hub

- **URL (B cells):** <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Bcells/>
- **URL (Monocytes):** <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Monocytes>
- **URL (NK cells):** <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_NKcells/>
- **URL (T cells):** <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT_Tcells>
- **Maintainer:** [Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI)](https://www.unimedizin-mainz.de/imbei/imbei/welcome-page.html?L=1)
- **Version:** iSEE {2.9.0}

**More:** Those apps are listed at <http://shiny.imbei.uni-mainz.de:3838/iSEE_covidIT/>.

## Upload your own SummarizedExperiment (CCB HMS)

- **URL:** <https://ccb-rstudio-connect.hms.harvard.edu/iSEE_enterprise/>
- **Maintainer:** [Center for Computational Biomedicine (CCB) Harvard Medical School](https://computationalbiomed.hms.harvard.edu/)
- **Version:** iSEE {2.4.0}

## Spatially resolved transcriptomics of the inferior temporal cortex (pseudobulks)

- **URL:** <https://libd.shinyapps.io/Visium_SPG_AD_pseudobulk_AD_pathology_wholegenome/>
- **Maintainer:** [The Lieber Institute for Brain Development](https://www.libd.org/)
- **Version:** iSEE {2.10.0}

**More:** This app accompanies the article <https://doi.org/10.1101/2023.04.20.537710>

## Myocyte data sets

- **URL:** <https://shiny.mdc-berlin.de/MyoExplorer/>
- **Maintainer:** [Max Delbrück Center](https://www.mdc-berlin.de/)
- **Version:** iSEE {2.6.0}

## Single-nucleus transcriptome analysis in human brain

- **URL:** <https://libd.shinyapps.io/tran2020_Amyg/>
- **URL:** <https://libd.shinyapps.io/tran2021_NAc/>
- **URL:** <https://libd.shinyapps.io/tran2021_sACC/>
- **URL:** <https://libd.shinyapps.io/tran2021_DLPFC/>
- **URL:** <https://libd.shinyapps.io/tran2021_AMY/>
- **URL:** <https://libd.shinyapps.io/tran2021_HPC/>
- **Maintainer:** [The Lieber Institute for Brain Development](https://www.libd.org/)
- **Version:** iSEE {2.4.0}

**More:** Those apps accompany the article <https://www.cell.com/neuron/fulltext/S0896-6273(21)00655-3>

## HI-TEAM (HIV-Infected cell Transcriptome and EpitrAnscriptoMe)

- **URL:** <https://hi-team.unil.ch/#>
- **Maintainer:** [Ciuffi Lab](https://www.chuv.ch/en/microbiologie/imu-home/research/research-groups/angela-ciuffi)
- **Version:** iSEE {2.1.10}, iSEEu {1.1.4}

**More:** This app accompanies the article <https://www.frontiersin.org/articles/10.3389/fviro.2021.714475/full>.\

## Comparison of SARS-CoV-1 and SARS-CoV-2 

- **URL:** <https://shiny.mdc-berlin.de/COV/>
- **Maintainer:** [Max Delbrück Center](https://www.mdc-berlin.de/)
- **Version:** iSEE {1.6.1}

## CD8 TILs from B16 melanoma tumors

- **URL:** <https://tilatlas.shinyapps.io/B16_CD8TIL_10X/>
- **Maintainer:** [Swiss Portal for Immune Cell Analysis](https://spica.unil.ch/)
- **Version:** iSEE {1.2.4}

## Demonstration iSEE apps (2018)

- **URL (single-cell RNA-seq, mouse visual cortex):** <https://marionilab.cruk.cam.ac.uk/iSEE_allen>
- **URL (RNA-seq, The Cancer Genome Atlas):** <https://marionilab.cruk.cam.ac.uk/iSEE_tcga>
- **URL (single-cell RNA-seq, peripheral blood mononuclear cells):** <https://marionilab.cruk.cam.ac.uk/iSEE_pbmc4k>
- **URL (mass cytometry, healthy and diseased human donors):** <https://marionilab.cruk.cam.ac.uk/iSEE_cytof>
- **Maintainer:** [Marioni group](https://www.cruk.cam.ac.uk/research-groups/marioni-group)
- **Version:** iSEE {1.2.4}

**More:** Those apps are listed at <https://marionilab.cruk.cam.ac.uk/>, among other interactive apps from the same developer.
